ticTacToe.rb

```diff
@@ -49,31 +32,14 @@ def get_player_input
     gets.chomp.to_s
   end
 
-  def is_user_input_valid(choice)
-    if choice !~ /\D/ && choice.length > 0
-      num = choice.to_i
-      if num <= 9 && num >= 1
-        true
-      else
-        false
-      end
-    else
-      false
-    end
-  end
-
   def is_space_free(choice)
     num = choice.to_i
     choice == @board[num - 1]
   end
 
-  def mark_board(choice)
-    @board[choice.to_i - 1] = get_marker if is_user_input_valid(choice) && is_space_free(choice)
-  end
-
   def move_confirmation(choice)
     if !is_user_input_valid(choice)
-      puts "Sorry that is not a valid move. Please select an integer between 0 and 8.\n"
+      puts "Sorry that is not a valid move. Please select an integer between 1 and 9.\n"
```

Comment:
Nice! I like this change!

Reply:
Thanks! Sunny caught this for me - I have to do a better job proof reading!